br/tests/br_pitr_failpoint/check/check_ingest_repair.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright 2024 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# check index schema
+## check table test.pairs
+run_sql "SHOW INDEX FROM test.pairs WHERE Key_name = 'i1';"
+check_contains "Column_name: y"
+check_contains "Column_name: z"
+
+# check index data
```

Comment:
also ADMIN CHECK TABLE to find data-index inconsistency

Reply:
done